### DISCUSSION TITLE
Display number of discussions on the tab

### DISCUSSION THREAD

**Discussion by mculp** [community/community#3580] [Discussions]
It'd be great if the tab showed the number of discussions on the tab, similar to Pull Requests and Issues. For example, this would show that we have 5 discussions:

<img width="537" alt="Screen Shot 2021-03-18 at 4 42 58 PM" src="https://user-images.githubusercontent.com/129983/111701608-15fe4280-8809-11eb-99a9-fc829ac9862d.png">

It'd be even better if there was a way to show _unread_ discussions, so a user would know when there are new unread discussions.

The main problem without having a number is that the discussions tab tends to blend in with the other tabs that are not clicked on as frequently. If there's a number displayed, it's kind of a call to action.

**Comment by EricThomson** [comment]
See  #48750 -- Tried and disabled.
